bundles/org.openhab.binding.shelly/src/main/resources/ESH-INF/thing/relay.xml

```diff
@@ -284,4 +262,217 @@
 			<channel id="powerLed" typeId="ledPowerDisable" />
 		</channels>
 	</channel-group-type>
+	<channel-group-type id="externalSensors">
+		<label>External Sensors</label>
+		<description>Temperatures from external sensors connected to the optional Addon</description>
+	</channel-group-type>
+	<channel-type id="timerAutoOn" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Auto-ON Timer</label>
+		<description>ON: After the output was turned off it turns on automatically after xx seconds; 0 disables the timer</description>
+		<state min="0" step="1" pattern="%d %unit%" readOnly="false"></state>
+	</channel-type>
+	<channel-type id="timerAutoOff" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Auto-OFF Timer</label>
+		<description>ON: After the output was turned on it turns off automatically after xx seconds; 0 disables the timer</description>
+		<state min="0" step="1" pattern="%d %unit%" readOnly="false"></state>
+	</channel-type>
+	<channel-type id="timerActive" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Auto ON/OFF timer active</label>
+		<description>ON: A timer is active, OFF: no timer active</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="rollerShutter">
+		<item-type>Rollershutter</item-type>
+		<label>Roller Control (0=open, 100=closed))</label>
+		<description>Controls the roller</description>
+		<category>Blinds</category>
+	</channel-type>
+	<channel-type id="rollerPosition" advanced="true">
+		<item-type>Dimmer</item-type>
+		<label>Roller Position (100=open, 0=closed)</label>
+		<description>Position the roller (100..0 in %, where 100%=open, 0%=closed)</description>
+		<state readOnly="false" min="0" max="100" />
+	</channel-type>
+	<channel-type id="rollerStop">
+		<item-type>String</item-type>
+		<label>Roller stop reason</label>
```

Comment:
The first letter of every word in a label should be capitalized.
```suggestion
		<label>Roller Stop Reason</label>
```

Reply:
changed